src/panel/poi/blocks/StarsBlock.jsx

```diff
@@ -30,7 +30,7 @@ const StarsBlock = ({ block, subclass }) => {
 
   return (
     <Block simple icon={<IconStar fill={ACTION_BLUE_BASE} width={20} />}>
-      {_('Starred {subclass}', 'poi', { subclass })}
+      {capitalizeFirst(_('Starred {subclass}', 'poi', { subclass }))}
```

Comment:
As I understand it, it will have no effect with the English string, but would enforce capitalization of the first word for other languages. Shouldn't we just expect the translations to be consistent with the capitalization of the reference?

Reply:
indeed, for this component, it can be handled entirely in the translations

Reply:
In this precise string I belive we have to force the "capitalize first".
Ex: in french we will say "{subclass} étoilé", and in english "Starred {subclass}"
subclass is lowercase in every case.
So when subclass = "restaurant" for example,
we need a way to have: FR: "Restaurant étoilé" (with a R) / EN: "Starred restaurant" (with a r)
IMO only capitalizeFirst (or CSS tricks) can achieve this, but not the translation.

Reply:
I completely missed that case.

This is typically a case where I show that I'm a novice regarding this kind of issues. Nice catch!